### ISSUE TITLE
crash v1.3.8

### ISSUE THREAD

**Issue by ACIIL** [defeatedcrow/HeatAndClimateMod#13]
Crash may connect to Forestry. I'm using forestry{5.2.17.368} [Forestry] (forestry_1.10.2-5.2.17.368.jar).
Full crash log available upon request.

```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from HeatAndClimateMod (dcs_climate)
Caused by: java.lang.IllegalAccessError: tried to access field forestry.api.farming.Farmables.fertilizers from class defeatedcrow.hac.plugin.DCPluginForestry
	at defeatedcrow.hac.plugin.DCPluginForestry.load(DCPluginForestry.java:201)
	at defeatedcrow.hac.plugin.DCIntegrationCore.load(DCIntegrationCore.java:40)
	at defeatedcrow.hac.main.ClimateMain.postInit(ClimateMain.java:105)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:616)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:243)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:221)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at com.google.common.eventbus.EventSubscriber.handleEvent(EventSubscriber.java:74)
	at com.google.common.eventbus.SynchronizedEventSubscriber.handleEvent(SynchronizedEventSubscriber.java:47)
	at com.google.common.eventbus.EventBus.dispatch(EventBus.java:322)
	at com.google.common.eventbus.EventBus.dispatchQueuedEvents(EventBus.java:304)
	at com.google.common.eventbus.EventBus.post(EventBus.java:275)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:145)
	at net.minecraftforge.fml.common.Loader.initializeMods(Loader.java:810)
	at net.minecraftforge.fml.client.FMLClientHandler.finishMinecraftLoading(FMLClientHandler.java:334)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:520)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:351)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
```

**Comment by defeatedcrow** [comment]
Thank you for your report!

Currently I am using the Forestry API version 5.2.17.360. Perhaps this is too old.
I will update my environment. Please wait a moment.

**Comment by ACIIL** [comment]
Current forestry 1.10.2 file:
https://minecraft.curseforge.com/projects/forestry/files?filter-game-version=2020709689%3A6170

**Comment by defeatedcrow** [comment]
Thank you!

**Comment by defeatedcrow** [comment]
It fixed in 3f61412
